[2024-06-20T21:30:45.059+0000] {scheduler_job_runner.py:769} ERROR - Executor reports task instance <TaskInstance: example_bash_operator.this_will_skip scheduled__2024-06-19T00:00:00+00:00 [queued]> finished (failed) although the task says it's queued. (Info: None) Was the task killed externally?
